### PR TITLE
chore: prepare v0.10.5

### PR DESCRIPTION
**Pull request by daeMOn63**
- bump cosmos sdk to v0.45.6. The minimal  go version required is now 1.18.
- bump wasmd to v0.27
- bump ibc-go to v3.1.0

### CODE REVIEW

**Review by ejfitzgerald — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -671,27 +709,149 @@ func (app *App) GetSubspace(moduleName string) paramstypes.Subspace {
...
+			panic("mm.Modules[icatypes.ModuleName] is not of type ica.AppModule")
+		}
+		icamodule.InitModule(ctx, controllerParams, hostParams)
+
+		validDenoms := map[string]banktypes.Metadata{
+			"atestfet": {
+				Base:        "atestfet",
+				Description: "Fetch-ai Network testnet token",
+				DenomUnits: []*banktypes.DenomUnit{
+					{Denom: "TESTFET", Exponent: 18},
```
Comment (ianharris-fetch) [inline app/app.go:793]: Why no mtestfet, utestfet etc etc?

Comment (daeMOn63) [reply app/app.go:793]: Not sure why @ejfitzgerald added these to the `FET` one. I can add but is there any use for them?

Comment (ianharris-fetch) [reply app/app.go:793]: Not aware of any use for them myself.  Possibly worth keeping testnet and mainnet consistent though.

Comment (daeMOn63) [reply app/app.go:793]: Added anyway in #257 ;)

**Pull request merged**